components/eventing-controller/reconciler/subscription/reconciler.go

```diff
@@ -180,7 +177,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	// Save the subscription status if it was changed
 	if statusChanged {
 		if err := r.Status().Update(ctx, desiredSubscription); err != nil {
-			log.Error(err, "Update subscription status failed")
+			log.Errorw("update subscription status failed", "error", err)
```

Comment:
Do you wanna add a reconciled successfully log with the subscription name?

Reply:
Added.